doc/dash/dash-sonic-hld.md

```diff
@@ -89,6 +92,7 @@ At a high level the following should be supported:
     - Telemetry and Monitoring
     - Private Link
     - Private Link NSG
+    - Express Route GW Bypass
```

Comment:
can we have a config example for express route gw bypass ?

Reply:
Will add as a next iteration (another PR)

Reply:
added an example